Modules/_winapi.c

```diff
@@ -1512,6 +1512,63 @@ _winapi_PeekNamedPipe_impl(PyObject *module, HANDLE handle, int size)
     }
 }
 
+/*[clinic input]
+_winapi.LCMapStringEx
+
+    locale: LPCWSTR
+    flags: DWORD
+    src: LPCWSTR
+
+[clinic start generated code]*/
+
+static PyObject *
+_winapi_LCMapStringEx_impl(PyObject *module, LPCWSTR locale, DWORD flags,
+                           LPCWSTR src)
+/*[clinic end generated code: output=cf4713d80e2b47c9 input=9fe26f95d5ab0001]*/
+{
+    if (flags & LCMAP_SORTHANDLE) {
+        PyErr_SetString(PyExc_ValueError, "LCMAP_SORTHANDLE is not supported");
+        return NULL;
+    }
+    if (flags & LCMAP_HASH) {
+        PyErr_SetString(PyExc_ValueError, "LCMAP_HASH is not supported");
+        return NULL;
+    }
+    if (flags & LCMAP_BYTEREV) {
+        PyErr_SetString(PyExc_ValueError, "LCMAP_BYTEREV is not supported");
+        return NULL;
+    }
+    if (flags & LCMAP_SORTKEY) {
+        PyErr_SetString(PyExc_ValueError, "LCMAP_SORTKEY is not supported");
+        return NULL;
+    }
```

Comment:
Does it really matter to say exactly which flag isn't supported? If not, I'd simply use the following:

```suggestion
    if (flags & (LCMAP_SORTHANDLE | LCMAP_HASH | LCMAP_BYTEREV |
                 LCMAP_SORTKEY)) {
        return PyErr_Format(PyExc_ValueError, "unsupported flags");
    }
```